### ISSUE TITLE
Add a map displaying buildings for which we have applications

### ISSUE THREAD

**Issue by jrunningen** [codefordc/districthousing#14]
It could make for a better user interface if the user can see pins on a map for each building resource.

**Comment by meiao** [comment]
Hey guys.
I was thinking about working on this, but I hit 2 walls.

1) On my instance, there is no information about the buildings.
Is there an automated way to add them to the db?

2) The file at https://districthousing.org/housing_forms.json has no information about latitudes and longitudes.

Anyway, I was thinking on using [leaflet](http://leafletjs.com/) to make it.

**Comment by meiao** [comment]
Hey guys. I'll start to work on this.

**Comment by jrunningen** [comment]
Yes, we will have to attach a location to each PDF for it to be shown on a map. There are a couple of ways you can do this.  Probably the best way is to attach metadata to the PDFs in the formsync folder on Google drive.

Use these instructions, from the bottom of the make a PDF page:

Open your PDF in Acrobat.
Click File menu -> Properties...
Click the Custom tab.
In the "Name" field, type "Location".
In the "Value" field, enter the address of this housing location. This should be something that Google Maps could find your building with.
Click "Add".
Click "OK".
Save your PDF.

**Comment by meiao** [comment]
Hi guys. I just made a commit adding something similar to this.
What I made was to add a button on the location of any housing that has its lat and long fields filled.
When this button is clicked it shows a modal with the map, centered at the housing, with a marker showing it.
I made some comments on the code. And I would appreciate any comment.

I can get things done in Rails, but I am not sure of the best ways to do it.
